packages/dx-react-scheduler/src/plugins/today-button.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { getMessagesFormatter } from '@devexpress/dx-core';
+import {
+  Plugin,
+  Template,
+  TemplatePlaceholder,
+  TemplateConnector,
+  PluginComponents,
+} from '@devexpress/dx-react-core';
+import { TodayButtonProps } from '../types/today-button';
```

Comment:
We can import this interface from index file: `.. from '../types'` so that future type imports will stick together